tests/unittests/run_tests.py

```
@@ -190,6 +192,155 @@ def test_run_timestamps(self):
+            "runs",
+        ):
+            self.assertIsInstance(
+                fields[name], food._ConnectedReferencesDictField, name
+            )
+
+        for name in ("saved_views", "workspaces"):
+            self.assertIsInstance(
+                fields[name], food._ConnectedReferencesListField, name
+            )
+
+    def _disconnect(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_disconnect`

Add return type annotation: `None`

(ANN202)